all/src/main/content/META-INF/vault/filter.xml

```diff
@@ -19,7 +19,6 @@
 
 <workspaceFilter version="1.0">
     <filter root="/apps/acs-commons/config"/>
-    <filter root="/apps/acs-commons/config.rde"/>
```

Comment:
I would rather leave that entry with `type="cleanup"` to make sure this node is removed again on 6.5.x/AEM Cloud SDK servers having previously installed 6.0.4.